kotlin/add-graphics-renderer/src/main/java/com/esri/arcgisruntime/sample/addgraphicsrenderer/MainActivity.kt

```diff
@@ -19,15 +19,7 @@ import android.graphics.Color
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
 import com.esri.arcgisruntime.ArcGISRuntimeEnvironment
-import com.esri.arcgisruntime.geometry.CubicBezierSegment
-import com.esri.arcgisruntime.geometry.EllipticArcSegment
-import com.esri.arcgisruntime.geometry.Geometry
-import com.esri.arcgisruntime.geometry.Part
-import com.esri.arcgisruntime.geometry.Point
-import com.esri.arcgisruntime.geometry.Polygon
-import com.esri.arcgisruntime.geometry.PolygonBuilder
-import com.esri.arcgisruntime.geometry.PolylineBuilder
-import com.esri.arcgisruntime.geometry.SpatialReferences
+import com.esri.arcgisruntime.geometry.*
```

Comment:
It can be instructive to leave these with their specific imports

Reply:
Added them in! There was quite a few of imports :)